cdap-ui/app/features/dashboard/controllers/addwdgt-ctrl.js

```diff
@@ -65,7 +65,9 @@ function ($scope, $modalInstance, caskFocusManager, Widget) {
           new Widget({
             type: $scope.model.type,
             title: $scope.model.metric.title,
-            color: classes[(generateHash(value) * 13) % classes.length],
+            settings: {
+              color: classes[(generateHash(value) * 13) % classes.length]
```

Comment:
As discussed this is being used in overview-ctrl and we can have this option open if the user wants to have a custom color for the chart.